3d_segmentation/swin_unetr_brats21_segmentation_3d.ipynb

```diff
@@ -0,0 +1,1021 @@
+{
```

Comment:
<p>Could you please help provide the python script to generate this datalist from original data? Instead of downloading from the Google Drive.</p><p>I think the script may be more useful for users to adapt to other datasets.</p><p> </p><p>Thanks.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/750/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Hi @Nic-Ma,

I did not generate this split as it was being used internally by multiple team members. I suppose generating data split is not specific to this tutorial ? if ok, let's skip this one.

Reply:
OK, MONAI Google Drive is acceptable so far.

Thanks.